src/timetable_gui.cpp

```diff
@@ -571,7 +630,9 @@ struct TimetableWindow : Window {
 
 				if (order != nullptr) {
 					uint time = (selected % 2 != 0) ? order->GetTravelTime() : order->GetWaitTime();
-					if (!_settings_client.gui.timetable_in_ticks) time /= Ticks::DAY_TICKS;
+					if (_settings_client.gui.timetable_mode != TimetableMode::Ticks) {
+						time /= TicksPerDayOrSecond();
+					}
```

Comment:
Special-case that could be removed if `TicksPerDayOrSecond()` also handled `TimetableMode::Ticks`.